.github/workflows/docs.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         make docs
 
     - name: Configure AWS Credentials
-      if: ${{ github.event_name == 'push' }}
+      if: ${{ github.event_name == 'push' && ( github.ref_name == 'dev' || startsWith(github.ref_name, 'v0.') }}
```

Comment:
This will hit us back when we release `v1.0.0`

Reply:
Who says we ever will?!